### DISCUSSION TITLE
Individual proxy settings is needed

### DISCUSSION THREAD

**Discussion by zbutfly** [TumblThreeApp/TumblThree#394] [General]
I know the app use windows system proxy settings, but individual settings of proxy for http/socks5, plain and auth are essential。

**Comment by thomas694** [comment]
Why are they essential and needed?

**Reply by zbutfly** [reply]
At my office, I need a proxy to access tumblr and another main proxy to access intranet. for working... only for an example.

And, the windows proxy setting for http proxy with auth, looks not work correctly for the .net application. I had tried it and not succeeded (not confirmed the issue caused the system proxy settings,  but MS officially confirmed the OneDrive application does not supports auth http proxy settings from system).

**Comment by thomas694** [comment]
Honestly, after the first three words I wanted to stop reading on...
The app isn't made for circumventing anything, but for normal users in their private environment where the vast majority of users don't have a proxy anyway.

The system proxy should work, but maybe not for your use case.
Until there arises need from a broader user base, we won't put any effort into it, except supporting a volunteer who steps up.

**Reply by pixelqiwi** [reply]
Honestly, you sound like you don't want to help people use your app. 

In my country, Tumblr is only available through a proxy, which kind of ruins the point of using this app, when it literally can't connect to the website simply because it gives me a 407 and I don't even have an option to feed it the proxy and its credentials directly. And it doesn't seem to recognise the credentials from Windows' Credential Manager. The software simply becomes unusable for me because of this.

Oh, and not to say that Windows doesn't let you use SOCKS proxies at all.

In my honest opinion, proxy settings should be in every program that connects to the internet. You never know who might use it, but people will use it.

**Reply by thomas694** [reply]
Give this topic a thumbs up, if not done yet.

Maybe in the meantime, someone can give an advice and workaround how to get it running with an authenticated proxy server.
